application/helpers/qanda_helper.php

```diff
@@ -3165,7 +3180,7 @@ function do_multiplenumeric($ia)
                     $answer_main .= $dispVal;
                 }
 
-                $answer_main .= '" onkeyup="'.$checkconditionFunction.'(this.value, this.name, this.type);" '." {$maxlength} />\n\t".$suffix."\n</span>{$sliderright}\n\t</li>\n";
+                $answer_main .= '" onkeyup="'.$checkconditionFunction.'(this.value, this.name, this.type, \'onchange\','.$integeronly.');" '." {$maxlength} />\n\t".$suffix."\n</span>{$sliderright}\n\t</li>\n";
```

Comment:
It's another issue here ? ANd i think we must remove all inline javascript ... too complicated to manage in template or in plugin.